app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="user_holder">
+  <h2>Log in</h2>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.email_field :email, placeholder:"Email", autofocus: true, autocomplete: "email" , class:'user_input' %>
+    </div>
+    <br>
+    <div class="field">
+      <%= f.password_field :password, placeholder:"Password", autocomplete: "current-password", class:'user_input' %>
+    </div>
+    <br>
+    <% if devise_mapping.rememberable? %>
+      <div class="field user_input"  >
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+      <br>
+    <% end %>
+    <div class="actions">
+      <%= f.submit "Log in",  class:'user_input' %>
+    </div>
+  <% end %>
+  <div class="user_setting">
+    <%= render "devise/shared/links" %>
+  </div>
+</div>
```

Comment:
- [ ] Please style the registration and login page to match the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)
![image](https://github.com/Donmark2k/blog_app/assets/55596970/e6ca6366-d4a1-489b-b8cf-767a73307c23)

Reply:
@CollinsTatang, If I match this wireframe, how will a new user register?

Reply:
@CollinsTatang , I have resolved it